creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
 ### Minecraft: Bedrock Edition
 
-Bedrock Edition was launched on Sept. 20, 2017 and was based on Minecraft: Pocket Edition, which was released in 2011. It brought together nine of the major device platforms under a singular codebase called the Bedrock Engine. This was a rewrite of Minecraft from the ground up and brought along with it some fundamental changes to the platform paving the way for an exciting new development community. This edition is commonly referred to as simply ***Bedrock***.
+Bedrock Edition was fully launched in September, 2017, based on Minecraft: Pocket Edition, which was released in 2011. It brought together nine of the major device platforms under a singular codebase called the Bedrock Engine. This was a rewrite of Minecraft from the ground up using the [C++ programming language](https://en.wikipedia.org/wiki/C%2B%2B). It's available on Windows 10 PCs, Xbox, PlayStation, Nintendo Switch, Android, and iOS. This edition is commonly referred to as simply ***Bedrock***.
 
-## World Differences
+## World Format Differences
 
-The most obvious difference between both versions is the world format. Bedrock Edition uses the LevelDB format for world storage while Java Edition uses the Anvil format. Due to this, most third-party tools created for Java Edition world editing will not work on Bedrock Edition.
+One major technical difference between both versions is the world format. Bedrock Edition uses Google's [LevelDB format](https://en.wikipedia.org/wiki/LevelDB) for world storage, while Java Edition uses a custom [Anvil format](https://wiki.vg/Map_Format). Due to this, most third-party tools created for world editing in a specific edition will not work on the other.
 
-The two versions also use a fairly different block format. Java Edition has flattened its block format using a unique string for each individual block and storing the state of that block separately. Similarly, Bedrock Edition has moved to a string-based system with block states, but have kept some blocks grouped together defined by data value. Basically, this means that blocks are named differently between the versions. In Bedrock Edition, granite would be `stone 1` whereas on Java Edition it’s simply `granite`.
+Many blocks and items are also named differently between the two versions. In 1.13, Java Edition underwent an internal [flattening](https://minecraft.fandom.com/wiki/Java_Edition_1.13/Flattening), removing the concept of "damage values" so that each block has its own name. Bedrock has not made this change, and so still has the pre-1.13 block names. For example, granite is `granite` in Java Edition, and `stone:1` in Bedrock Edition.
 
```

Comment:
Changed the last line of the first paragraph because although historically there have definitely been a lot of Java-only tools, I feel like that's starting to change and now you might be more likely to find Bedrock-specific tools. Anyway either way the new line doesn't pick sides.
And I rewrote the second paragraph because it implied that Bedrock's block state system is its own idea and not just exactly what Java was doing back in 1.12.

Reply:
We cannot link to third party sites.

Reply:
I see now that Microsoft's rules for its documentation sites make a comprehensive ecosystem of information impossible. The information on that page, and many other select wiki pages, is extremely helpful and well-maintained, and its production values vastly exceed those of any of the articles in this repository. Without the ability to link to it, the only option is for this repo to host its own equally comprehensive and visually well-organized page, which will obviously never happen.

That's a shame. I know it's not your fault, but I hope Microsoft is aware that their policy to refuse acknowledging better sources of information only makes them less helpful as docs. It's fortunate that in this case, the concept is merely tangential, but giving readers the ability to find out more about an unfamiliar concept with a link is an undervalued property of good docs.

Do whatever you like with my changes, I think it's unlikely that I'll be contributing further to a repo whose rules are in conflict with its goals.